notebooks/icrp107_dataset/icrp107_dataset.ipynb

```diff
@@ -1257,14 +1427,17 @@
     "            b = np.unique(np.concatenate((b,rows_dict[j])))\n",
     "    rows_dict[i] = b\n",
     "\n",
-    "rows_c = np.array([], dtype=np.int)\n",
-    "cols_c = np.array([], dtype=np.int)\n",
+    "rows_c = np.array([], dtype=np.int64)\n",
+    "cols_c = np.array([], dtype=np.int64)\n",
     "for i in range(0, num_nuclides):\n",
+    "    # row of C initialized as array of decay chain members\n",
     "    rows_c = np.concatenate((rows_c,rows_dict[i]))\n",
-    "    cols_c = np.concatenate((cols_c,np.array([i]*len(rows_dict[i]))))\n",
+    "    # column of C initialized as array of index with length of chain size\n",
+    "    cols_c = np.concatenate((cols_c,np.array([i]*len(rows_dict[i]), dtype=np.int64)))\n",
     "\n",
-    "c = sparse.csc_matrix((np.array([0.0]*rows_c.size, dtype=np.float64), (rows_c, cols_c)))\n",
-    "c_inv = sparse.csc_matrix((np.array([0.0]*rows_c.size, dtype=np.float64), (rows_c, cols_c)))"
+    "# initialize c and c^(-1) as sparse csc matrices with ones on diagonal\n",
+    "c = sparse.csc_matrix(sparse.eye(rows_c.size, cols_c.size, dtype=np.float64))\n",
+    "c_inv = sparse.csc_matrix(sparse.eye(rows_c.size, cols_c.size, dtype=np.float64))"
```

Comment:
I really like the switch to using eye! It looks like you can specify `format="csc"` as a parameter to `sparse.eye()`, which would save doing the calls to `sparse.csc_matrix()`.
https://docs.scipy.org/doc/scipy/reference/generated/scipy.sparse.eye.html